src/buy/index.ts

```diff
@@ -0,0 +1,2 @@
+// 🌲☀🌲
+export { Buy } from './components/Buy';
```

Comment:
Make sure to add to exports in package.json as well

Reply:
this component doesnt have children so i think this is only export i need

Reply:
i put the exports in playground PR